frontend/src/app/AppRoutes.tsx

```diff
@@ -128,6 +131,9 @@ const AppRoutes: React.FC = () => {
             {isConnectionTypesAvailable ? (
               <Route path="/connectionTypes/*" element={<ConnectionTypeRoutes />} />
             ) : null}
+            {isStorageClassesAvailable && (
+              <Route path="/storageClasses/*" element={<StorageClassesPage />} />
+            )}
             <Route path="/modelRegistrySettings/*" element={<ModelRegistrySettingsRoutes />} />
```

Comment:
dont prevent users from accessing this page, just hide the nav item. follow how the other admin routes are.

I do see ConnectionTypes is like this, but it shouldn't be. unless Andrew is changing the way he wants the flags to work recently

Reply:
Seemed to make sense to me to not allow even admins to view the page if the feature isn't ready, but I can remove the condition.

Reply:
More of just a consistency things. I tried to do it the way you had before and was told to not disable the route